src/main/java/echoMsg.java

```diff
@@ -2,6 +2,7 @@
 /* 8-31-21 */
 public class echoMsg  {
     private static String userName = "USERNAME";
+    private static String aruguments = "Here are the arguments you passed in.";

```

Comment:
Is this the right way to pull the string literals?

Reply:
To me this looks like the best way.